lib/page/prompt.js

```diff
@@ -137,6 +192,11 @@ function promptSubmit() {
 		case "keybind":
 			data = dataElement.submit();
 			break;
+		case "multiInput":
+			data = [];
+			const x = document.querySelectorAll("#data");
+			for (let i = 0; i < x.length; i++) { data.push(x[i].value); }
+			break;
```

Comment:
could be minimized to
`data = document.querySelectorAll("#data").map(element => element.value)`